published/090-2020-may-19.md

```diff
@@ -0,0 +1,43 @@
+# Welcome to the IPFS Weekly
+
+Here’s what’s happening lately in the [InterPlanetary File System](https://ipfs.io/) galaxy and beyond! 🚀
+
+## The path to decentralization and go-ipfs 0.5 on the Zero Knowledge Podcast
+IPFS Project Lead Molly Mackinlay was recently a guest on the [Zero Knowledge Podcast](https://www.zeroknowledge.fm/129) talk about her experience guiding the IPFS project, as well as libp2p, Bitswap, Testground, the use cases and projects working with these tools, and the specific upgrades made with the recent 0.5 IPFS release. Enjoy!
+
+## Experience the IPFS Pinning Summit, again!
+True, the IPFS Pinning Summit has come and gone, but if you missed it, we have you covered. [Watch all of the videos](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTvsxuZ9T-tMlu_v0lja6v5) from the event and be sure to [read our recap](https://blog.ipfs.io/2020-05-13-ipfs-pinning-summit-recap/) from the two-day summit. 
+
+## Are you using (or plan to use) js-ipfs in browser?
+Then we want to hear from you! Your input would really help to shape the plan for changes that we hope will improve your and your users' experience. Check out the discussion on [reducing js-ipfs overhead by sharing node across browser tabs](https://discuss.ipfs.io/t/reducing-js-ipfs-overhead-by-sharing-node-across-browser-tabs/8024) now!
```

Comment:
Hey @Gozala let me know if this blurb reads okay. Thanks

Reply:
@renrutnnej This is great thanks!